bin/pulsar

```diff
@@ -70,7 +71,8 @@ where command is one of:
     broker              Run a broker server
     bookie              Run a bookie server
     zookeeper           Run a zookeeper server
-    global-zookeeper    Run a global-zookeeper server
+    global-zookeeper    Run a configuration-store server(Deprecated. Use "bin/pulsar configuration-store" instead)
+    configuration-store    Run a configuration-store server
```

Comment:
The documentation on this is out of line with the rest. I would suggest moving the rest into line with this.